### PR TITLE
Added first page

### PR DESCRIPTION
**Pull request by Moathal**
- Added README and MIT with their details.
- Created index.html file with toolbar header and headline section.
    1.    header (toolbar):
           - span (logo).
           - button.
    2.   section (headline):
          - h1 (mainTxt).
          - h3 (spprtTxt).
         
- Added CSS file flex layout and styling for each element with color and background, font, and sizing.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital@1&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Mo&lsquo;athal&lsquo;s Portfolio</title>
+</head>
+
+<body>
+    <div class="container flex">
+        <header class="toolbar">
+            <span class="theme logo"> Mo&lsquo;athal S. Kachi</span>
```
Comment (Lucas-Masaba) [inline index.html]: - [ ] Please wrap this with an `a` tag since it is a logo 👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            <button></button>
```
Comment (Lucas-Masaba) [inline index.html]: - [ ] Please wrap this hamburger button in a `nav` tag. The hamburger button contains navigation links to different places on your portfolio as you will see in the future 👌

File: index.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        </header>
+        <section class="headline flex">
+            <h1 class="mainTxt">Mo&lsquo;athal</h1>
+            <h3 class="spprtTxt">Software Developer</h3>
+        </section>
+    </div>
+</body>
+
+</html>
```
Comment (Lucas-Masaba) [inline index.html]: - [ ] You are missing a great deal of content. 
### Currently, your page looks like this
image
### It should look like this
image

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @Moathal  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 
- Clean code 🧹 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+                    <div class="frame">
+                        <h4>Skills</h4>
+                        <span>Codekit</span>
+                        <span>Github</span>
+                        <span>Gitlab</span>
+                        <span>Codepen</span>
+                        <span>Terminal</span>
+                    </div>
+                </div>
+
```
Comment (danmainah) [inline index.html]: - [ ] @Moathal great work implementing the changes requested, however, there is a small mistake on your languages and frameworks, ensure the content is aligned as in the given design:
image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Moathal , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @Moathal        , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

[OPTIONAL] Kindly add a screenshot to your README file showing your webpage user Interface. You can use extentions like fireshot to capturer the entire page.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**